cmd/metrics-v2.go

```diff
@@ -2380,7 +2380,7 @@ func getWebhookMetrics() *MetricsGroup {
 	}
 	mg.RegisterRead(func(ctx context.Context) []Metric {
 		tgts := append(logger.SystemTargets(), logger.AuditTargets()...)
-		metrics := make([]Metric, 0, len(tgts))
+		metrics := make([]Metric, 0, len(tgts)*4)
 		for _, t := range tgts {
```

Comment:
It doesn't really matter here, this is just an optimization.

Eventual slice could be always bigger. Not sure where the 4* comes here.

So this change is not needed.